pkg/webreg/webreg.go

```diff
@@ -734,15 +734,78 @@ syntax highlight all commands as bash.
 </p>
 
 <p>
-Sharing files between steps is supported via a shared directory. All
-containers will have an environment variable <code>SHARED_DIR</code> which
-contains the path of the shared directory. An artifacts directory also exists
-for steps that produce artifacts. The path for the artifacts directory is
-stored in the <code>ARTIFACTS_DIR</code> environment variable.
+A reference may be referred to in chains, workflows, and <code>ci-operator</code> configs.
 </p>
 
+<h4 id="execution"><a href="#execution">Step Execution Environment</a></h4>
 <p>
-A reference may be referred to in chains, workflows, and <code>ci-operator</code> configs.
+While a step simply defines a set of commands to run in a container image,
+by virtue of executing within a <code>ci-operator</code> workflow, the commands
+have a number of speical considerations for their execution environment.
+The commands can expect a set of environment variables to exist that inform
+them of the context in which they run; commands in steps can communicate to
+other steps via a shared directory in their filesystem.
+</p>
+
+<h5 id="env"><a href="#env">Available Environment Variables</a></h5>
+<p>
+The following environment variables will be available to commands in a step: 
+</p>
+
+<table>
+  <tr>
+    <th>Variable</th>
+	<th>Definition</th>
+	<th>Present When</th>
+  </tr>
+  <tr>
+    <td><code>${SHARED_DIR}<code></td>
+	<td>Directory on the step's filesystem where file shared between steps can be read.</td>
+	<td>Always.</td>
+  </tr>
+  <tr>
+    <td><code>${CLUSTER_PROFILE_DIR}<code></td>
+	<td>Directory on the step's filesystem where credentials and configuration from the cluster profile are stored.</td>
+	<td>When the test as defined in a <code>ci-operator</code> configuration file sets a <code>cluster_profile</code>.</td>
+  </tr>
+  <tr>
+    <td><code>${KUBECONFIG}<code></td>
+	<td>Path to <code>system:admin</code> credentials for the ephemeral OpenShift cluster under test.</td>
+	<td>After an ephemeral cluster has been installed.</td>
+  </tr>
+  <tr>
+    <td><code>${RELEASE_IMAGE_INITIAL}<code></td>
+	<td>Image pull specification for the initial release payload used to install the ephemeral OpenShift cluster.</td>
+	<td>Always.</td>
+  </tr>
+  <tr>
+    <td><code>${RELEASE_IMAGE_LATEST}<code></td>
+	<td>Image pull specification for the current release payload used as an upgade target on the ephemeral OpenShift cluster.</td>
+	<td>When a cluster upgrade is configured by setting <code>upgrade: true</code> in the <code>ci-operator</code> configuration file.</td>
```

Comment:
Not applicable to multi-stage tests, it is always included.

Reply:
I think adding an explanation in terms of the point at which the two release images are created (i.e. before and after `images` are built and tagged to `stable`) would make it clearer.